packages/tabs-react/documentation/Tabs.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Tabs
+react: tabs-react
+scss: tabs
+group: knapper
+---
+
+import TabsExample, { tabsExampleCode } from "./TabsExample";
+import "@fremtind/jkl-tabs/tabs.css";
+
+<Ingress>En liste med tabs kan brukes til å bytte mellom innhold på en side.</Ingress>
```

Comment:
Savner svar på noen spørsmål i dokumentasjonen her. Det er sånt som vi kan utvide etter hvert, men jo mer vi får inn fra start jo bedre. Ting har en tendens til å bli liggende når fokuset flytter seg til noe annet 😅

Hvorfor skal en bruke tabs i stedet for andre layouts? Når bruker jeg faner i stedet for accordions f. eks.? Når bruker jeg faner og når bør ting være en egen side?

Reply:
Hva med
```suggestion
<Ingress>En liste med tabs kan brukes til å bytte mellom likestilt innhold på en side.</Ingress>
```
En interaksjonsdesigner er vel egentlig den rette til å svare på dette. Synes uansett at vi bør prøve å holde ingressen veldig konsis, og heller utdype under "Bruk". Jo lenger ned man dytter det første live-eksemplet jo lenger tid tar det før man får satt informasjonen i kontekst.

Reply:
Ja, enig at det ikke hører hjemme i ingresen, men skulle gjerne hatt litt utvidet dokumentasjon før vi sier oss ferdig med faner for denne gang.

Reply:
@wkillerud @piofinn @Murstam I Flyt bruker vi også "tabs" til filtrering på lister. F.eks på saksoversikter. Her bruker vi tabs på å filtrere på samme listen
![Screenshot 2021-11-04 at 13 52 06](https://user-images.githubusercontent.com/91268511/140318952-e5b808b7-2d18-48b2-9562-4917a9a4db52.png)
.

Reply:
Ville det ikke vært mer naturlig å bruke f. eks radio buttons og puttet det sammen de andre filtreringsmulighetene som ligger under?

Reply:
Det kunne vi også gjort, men vi bruker det også litt som navigasjon, og hurtigvalg (samt at listene er litt ulike). Har sjekket litt nå, og i Flyt bruker vi mye tabs som navigasjonselement også for å strukturere innholdet på siden. Da endrer også URLén seg. Sånn jeg forstår så kan man definere tabs inn i kategorier: modultabs og navigasjonstabs, men at det brukes til å strukturere innhold.

Reply:
Ok. Jeg kjenner jeg blir litt usikker på om én komponent bør brukes til så mye forskjellig? Jeg tenkte i utgangspunktet at det er en komponent som kan brukes til å bytte mellom innhold på en side 🤔

Reply:
Jeg tenker at eksemplet fra flyt fungerer dersom man tenker på det som _to forskjellige_ sakslister 😊 Det virker det jo også litt som de er, hvis de vises på forskjellig måte? Men da ville jeg _ikke_ sagt at tabs er med på å filtrere, det er det fortsatt skjemaelementene som gjør. M.a.o. synes jeg heller ikke vi bør skrive at filtrering er en oppgave for tabs

Reply:
Og nå innså jeg nettop en ting til i det du skrev @karimarkhus, du nevner "modultabs" og "navigasjonstabs," og da innser jeg at både @Murstam og andre i denne PRen har omtalt navigasjonen i portalen som tabs. Menyelementene i portalen er **ikke** tabs, de er lenker i en meny/sidenavigasjon.

Jeg er enig at stylingen likner, men de har en helt annen oppgave. Tabs styrer _alltid_ innholdet i en _avgrenset del_ av siden. Se f.eks. eksemplet fra W3C-siden under. Hvis det skillet er utydelig bør kanskje designet også justeres?
<img width="367" alt="Skjermbilde 2021-11-05 kl  09 14 13" src="https://user-images.githubusercontent.com/25739615/140479268-5b7a9fd9-b9e8-46e4-9347-78462269a819.png">